yarn.lock

```diff
@@ -3803,6 +3807,10 @@ url@^0.11.0:
     punycode "1.3.2"
     querystring "0.2.0"
 
+"url@https://github.com/webcomponents/URL":
```

Comment:
To completely remove a module with yarn you need to use yarn remove name-of-module/url I guess